sdcm/nemesis.py

```diff
@@ -4528,15 +4532,18 @@ def disrupt_run_unique_sequence(self):
             self.steady_state_latency()
             self.has_steady_run = True
         InfoEvent(message='StartEvent - start a repair by ScyllaManager').publish()
-        self.disrupt_mgmt_repair_cli()
-        InfoEvent(message='FinishEvent - Manager repair has finished').publish()
+        if self.cluster.params.get('use_mgmt') and self.cluster.params.get('use_cloud_manager'):
```

Comment:
should be `and not`

Reply:
the original condition: `if not self.cluster.params.get('use_mgmt') and not self.cluster.params.get('use_cloud_manager'):`
So here should be `or` i think

Reply:
yes

Reply:
fixed